src/layouts/Sidebar.tsx

```diff
@@ -35,14 +47,17 @@ function Sidebar(this: any) {
           <Button component={Link} to={"/create"} color="inherit">
             Create
           </Button>
-          <Button color="inherit">Log out</Button>
+          <Button component={Link} to={"/auth"} color="inherit">
+            Log out
+          </Button>
         </Box>
       </Drawer>
-      <Button onClick={handleClickOpen} color="inherit">
-        Open
+
+      <Button {...conditionalProps} color="inherit">
+        {userData?.email ? userData!.email : "Log in"}
```

Comment:
Shorter version: `{ userData?.email ?? "Log In" }` - will display "Log In" if `userData?.email` is `null` or `undefined`